sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/index.py

```diff
@@ -7,14 +7,12 @@
 # cspell:disable-next-line
 from azure.ai.ml._restclient.azure_ai_assets_v2024_04_01.azureaiassetsv20240401.models import Index as RestIndex
 from azure.ai.ml._utils._arm_id_utils import AMLAssetId, AMLNamedArmId
-from azure.ai.ml._utils._experimental import experimental
 from azure.ai.ml.constants._common import LONG_URI_FORMAT
 from azure.ai.ml.entities._assets import Artifact
 from azure.ai.ml.entities._assets._artifacts.artifact import ArtifactStorageInfo
 from azure.ai.ml.entities._system_data import RestSystemData, SystemData
 
 
-@experimental
```

Comment:
why removing this?

Reply:
I thought it might be somewhat overkill to have 3 warnings come up when a user does something like this: `ml_client.indexes.create_or_update(Index(...))`.

I'm somewhat unclear on what the expectation is for what are _all_ the things we need to mark as experimental. Let me know if it's also supposed to be on the Asset too.

Reply:
it should be on the asset as well otherwise it's treated as GA

Reply:
Resolved by https://github.com/Azure/azure-sdk-for-python/pull/35439/commits/a7ee82fe3e9062064c87c9513e0739748f69843c